stdlib/sys/_monitoring.pyi

```diff
@@ -0,0 +1,44 @@
+from collections.abc import Callable
+from types import CodeType, SimpleNamespace
+from typing import Any
+
+DEBUGGER_ID: int
+COVERAGE_ID: int
+PROFILER_ID: int
+OPTIMIZER_ID: int
+
+def use_tool_id(id: int, name: str) -> None: ...
+def free_tool_id(id: int) -> None: ...
+def get_tool(id: int) -> str | None: ...
+
+class events(SimpleNamespace):
```

Comment:
At runtime, this doesn't actually inherit from `SimpleNamespace`; it's actually an _instance_ of `SimpleNamespace`. There's no good way to annotate instances of `SimpleNamespace` currently (and probably never will be). I think we should do something like what we already do with `sys.implementation`, which is a very similar situation:

https://github.com/python/typeshed/blob/df05ff2f52a1c0aaff1196661c251875da0c9eb2/stdlib/sys.pyi#L190-L196

```suggestion
events: _events

class _events:
```